usagov/usagov-2021
# Usagov 2044 drupal 10.3 view suggestions

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2044

## Description

Applies a modified version of the patch from https://www.drupal.org/project/drupal/issues/2118743 so that view-related template suggestions are displayed if twig debugging is on.

I changed that patch becuase it removed the space between the list icon and the name of the template, causing lines to show up as changed when I diffed the outputs pre- and post-patch.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Enable twig debugging, show template suggestions.
- Navigate to one or more pages with a view, confirm the HTML comments list view- related template suggestions.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
